### PR TITLE
Alias "u2f" to "webauthn" and partially cleanup

### PR DESCRIPTION
**Pull request by codingllama**
Alias the "u2f" second factor mode to "webauthn", effectively sunsetting U2F in favor of WebAuthn.

The change effectively disables "U2F mode" server-side, making Teleport use WebAuthn instead. This is in line with our compatibility promise, as Teleport 8.x clients are already WebAuthn-capable (and thus have no problems talking to the cluster).

I have cleaned up a good chunk of U2F references in lib/web and lib/client, plus a few other places. Changes on lib/auth are just the necessary to get the tests back to good standing. There is more work to be done, but this seems enough for a single PR.

#10375

### CODE REVIEW

**Comment by codingllama** [comment]
Friendly ping @kimlisa @zmb3 ?

**Review by zmb3 — APPROVED** [review]
LGTM.

Is there anywhere in the docs where we should mention that U2F is no longer supported and that it's an alias for webauthn now?

File: lib/client/weblogin.go [hunk 1]
```diff
@@ -353,10 +348,10 @@ func SSHAgentLogin(ctx context.Context, login SSHLoginDirect) (*auth.SSHLoginRes
 	return out, nil
 }
 
-// SSHAgentMFALogin requests a MFA challenge (U2F or OTP) via the proxy. If the
-// credentials are valid, the proxy wiil return a challenge. We then prompt the
-// user to provide 2nd factor and pass the response to the proxy. If the
-// authentication succeeds, we will get a temporary certificate back.
+// SSHAgentMFALogin requests a MFA challenge via the proxy.
+// If the credentials are valid, the proxy wiil return a challenge. We then
```
Comment (kimlisa) [inline lib/client/weblogin.go:352]:
```suggestion
// If the credentials are valid, the proxy will return a challenge. We then
```

**Review by kimlisa — APPROVED** [review]

**Comment by codingllama** [comment]
> LGTM.
> 
> Is there anywhere in the docs where we should mention that U2F is no longer supported and that it's an alias for webauthn now?

Definitely, there are some U2F articles we should cleanup, plus we could use a passwordless guide too. It's on my TODO list for the near future - give me a shout if you prefer to have it sooner rather than later.

**Comment by codingllama** [comment]
Thanks, folks. Merging asap.

**Pull request merged**